addon/mixins/embedded-records.js

```diff
@@ -0,0 +1,3 @@
+import EmbeddedRecordsMixin from "ember-data/-private/serializers/embedded-records-mixin";
+
+export default EmbeddedRecordsMixin;
```

Comment:
The existing path is `serializers/embedded-records-mixin`. Any one have any preference if we should keep it in the `serializers` folder when we expose it publicly since this mixin is coupled to the existing serializers?